eng/pipelines/docindex.yml

```diff
@@ -40,6 +40,15 @@ jobs:
         displayName: Update Docs Onboarding
         condition: and(succeeded(), or(eq(variables['Build.Reason'], 'Schedule'), eq(variables['Force.MainUpdate'], 'true')))
 
+      - task: Powershell@2
+        inputs:
+          pwsh: true
+          filePath: eng/common/scripts/Update-DocsMsToc.ps1
+          arguments: >-
+            -DocRepoLocation $(DocRepoLocation)
+            -OutputLocation $(DocRepoLocation)/docs-ref-mapping/reference-unified.yml
```

Comment:
Not a blocker. Seems like we can define the `outputLocation` in code by knowing `DocRepoLocation` instead of passing the parameters.